### PR TITLE
Icon: Fix default icon color mismatch with default fill color

### LINKED ISSUE #46595: Initial icon color mismatch bug
The icon panel has an issue that the color is initially undefined, but the UI defaults it to green (while it shows grey)

### PR DESCRIPTION
**Pull request by nmarrs**
**What this PR does / why we need it**:
Update default color of icon in panel to match default fill color.

Before
Initial Issue

After
Screen Shot 2022-03-16 at 8 31 56 AM

**Which issue(s) this PR fixes**:

Fixes #46595

### CODE REVIEW

File: public/app/features/canvas/elements/icon.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ export const iconItem: CanvasElementItem<IconConfig, IconData> = {
 
     const data: IconData = {
       path,
-      fill: cfg.fill ? ctx.getColor(cfg.fill).value() : '#CCC',
+      fill: cfg.fill ? ctx.getColor(cfg.fill).value() : '#19730E',
```
Comment (ryantxu) [inline public/app/features/canvas/elements/icon.tsx:104]: :thinking: I think this only appears to fix things (does that mean it is fixed?!)

IIUC, the real issue is that the config is undefined when it starts so it falls back too a default value.  The UI does the same thing.  This changes makes them both default to the same (in dark mode) but I don't think it fixes the general issue that will keep coming up if we don't actually initialize the settings

Comment (nmarrs) [reply public/app/features/canvas/elements/icon.tsx]: Ah yes, on looking at this again it looks like we were just missing including a fill property on the root icon object in `model.gen`

**Comment by torkelo** [comment]
not  sure I understand this, the model should not by default contain hex colors, but colors from the visualization palette

File: public/app/plugins/panel/icon/models.gen.ts [hunk 2]
```diff
@@ -18,6 +18,9 @@ export const defaultPanelOptions: PanelOptions = {
         mode: ResourceDimensionMode.Fixed,
         fixed: 'img/icons/unicons/analysis.svg',
       },
+      fill: {
+        fixed: '#19730E' // 'dark-green' to match default fill color
```
Comment (torkelo) [inline public/app/plugins/panel/icon/models.gen.ts:22]: this should just read "green" which is the default green color. You can use the named colors here:
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/createVisualizationColors.ts#L137
 
This is what other panels use, when you use the color picker a hex color should be avoided (unless it's  manual color), but the palette is all named colors that map to different colors depending current theme

**Review by ryantxu — APPROVED** [review]

**Pull request merged**